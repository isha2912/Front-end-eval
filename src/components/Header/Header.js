import React from 'react';
import './Header.scss';

const Header = (() => (
  <div className="nav-bar">
    My
    <b> Record </b>
    shelf
  </div>
));

export default Header;
