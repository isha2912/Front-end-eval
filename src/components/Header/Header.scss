.nav-bar{
    background-color: #38497E;
    height: 1.5rem;
    padding : 0.5rem;
    color : white;
}