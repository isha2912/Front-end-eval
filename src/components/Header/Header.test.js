import { render } from '@testing-library/react';
import React from 'react';
import Header from './Header';

describe(Header.name, () => {
  test('Should match snapshot', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
