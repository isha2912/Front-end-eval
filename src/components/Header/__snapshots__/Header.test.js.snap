// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should match snapshot 1`] = `
<div>
  <div
    class="nav-bar"
  >
    My
    <b>
       Record 
    </b>
    shelf
  </div>
</div>
`;
